runtime/src/xcm_config.rs

```diff
@@ -138,16 +134,25 @@ pub type XcmRouter = WithUniqueTopic<(
     ChildParachainRouter<Runtime, XcmPallet, PriceForChildParachainDelivery>,
 )>;
 
+// zkVerify-EVM-Parachain
+pub const ZKV_EVM_PARA_ID: u32 = 1;
+// paratest, included in this repository
 pub const TEST_PARA_ID: u32 = 1599;
+
 parameter_types! {
-    pub const Acme: AssetFilter = Wild(AllOf { fun: WildFungible, id: AssetId(TokenLocation::get()) });
+    pub const TVFYAsset: AssetFilter = Wild(AllOf { fun: WildFungible, id: AssetId(TokenLocation::get()) });
     pub TestParaLocation: Location = Parachain(TEST_PARA_ID).into_location();
-    pub AcmeForTest: (AssetFilter, Location) = (Acme::get(), TestParaLocation::get());
-    pub const MaxAssetsIntoHolding: u32 = 64;
+    pub ZKVEvmParaLocation: Location = Parachain(ZKV_EVM_PARA_ID).into_location();
+    pub TVFYAssetForTest: (AssetFilter, Location) = (TVFYAsset::get(), TestParaLocation::get());
+    pub TVFYAssetForZKVEvm: (AssetFilter, Location) = (TVFYAsset::get(), ZKVEvmParaLocation::get());
+    pub const MaxAssetsIntoHolding: u32 = 1;
```

Comment:
Polkadot sets this to 64. In which occasions the Assets are getting trapped ?

Reply:
This is not strictly related to assets being trapped. Given the simple scenarios that we want to support initially, and given that we do not plan to have any assets other than the native token for the time being, I thought there was no point in having more than `1` here. This is not a strong point though, so if we want to stick to Polkadot config, and be ready for unexpected uses of xcm, it's fine by me.